src/components/error/GlobalErrorContainer.react.js

```diff
@@ -1,68 +1,66 @@
 import {connect} from 'react-redux';
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import Radium from 'radium';
-import { isEmpty } from 'ramda';
-import { revert, resolveError } from '../../actions';
-import GlobalErrorOverlay from './GlobalErrorOverlay.react';
-import serverErrorCSS from './werkzueg.css';
+import {isEmpty} from 'ramda';
+import {revert, resolveError} from '../../actions';
+import './werkzeug.css';
+import {DebugMenu} from './menu/DebugMenu.react';
 
 class UnconnectedGlobalErrorContainer extends Component {
-  constructor(props) {
-    super(props);
-  }
+    constructor(props) {
+        super(props);
+    }
 
-  resolveError(dispatch, type, myId) {
-    if (type === 'backEnd'){
-      dispatch(resolveError({ type }));
-      dispatch(revert());
-    } else {
-      dispatch(resolveError({ myId, type }));
+    resolveError(dispatch, type, myId) {
+        if (type === 'backEnd') {
+            dispatch(resolveError({type}));
+            dispatch(revert());
+        } else {
+            dispatch(resolveError({myId, type}));
+        }
     }
-  }
 
-  serverError(error) {
-    var newWin = open('error.html','werkzueg','height=1024,width=1280');
-    newWin.document.write(error.backEnd.errorPage);
-    var debugger_css = newWin.document.getElementsByTagName('link')[0];
-    debugger_css.parentNode.removeChild(debugger_css);
-    var style = newWin.document.createElement('style');
-    style.type = 'text/css';
-    style.innerHTML = serverErrorCSS;
-    newWin.document.getElementsByTagName('head')[0].appendChild(style);
-    newWin.document.close();
-  }
+    // serverError(error) {
+    //     var newWin = open('error.html', 'werkzeug', 'height=1024,width=1280');
+    //     newWin.document.write(error.backEnd.errorPage);
+    //     var debugger_css = newWin.document.getElementsByTagName('link')[0];
+    //     debugger_css.parentNode.removeChild(debugger_css);
+    //     var style = newWin.document.createElement('style');
+    //     style.type = 'text/css';
+    //     style.innerHTML = serverErrorCSS;
+    //     newWin.document.getElementsByTagName('head')[0].appendChild(style);
+    //     newWin.document.close();
+    // }
```

Comment:
Is there a reason to keep this around?

Reply:
Looking back at old code and reminiscing about times past, reminding ourselves how young we really are and how anyone of us can go at any second?